cypress-tests/cypress/e2e/RoutingTest/00000-PriorityRouting.cy.js

```diff
@@ -0,0 +1,211 @@
+      });
+    });
+
+    after("flush global state", () => {
+      cy.task("setGlobalState", globalState.data);
+    });
+    it("retrieve-mca", () => {
+      cy.ListMCAbyMID(globalState);
+    });
+
+    it("api-key-create-call-test", () => {
+      cy.apiKeyCreateTest(fixtures.apiKeyCreateBody, globalState);
+    });
+
+    it("customer-create-call-test", () => {
+      cy.createCustomerCallTest(fixtures.customerCreateBody, globalState);
+    });
+
+    it("add-routing-config", () => {
+      let data = utils.getConnectorDetails("common")["priorityRouting"];
+      let req_data = data["Request"];
+      let res_data = data["Response"];
+
+      let routing_data = [
+        {
+          connector: "stripe",
+          merchant_connector_id: globalState.get("stripeMcaId"),
+        },
+        {
+          connector: "adyen",
+          merchant_connector_id: globalState.get("adyenMcaId"),
+        },
+      ];
+      cy.addRoutingConfig(
+        fixtures.routingConfigBody,
+        req_data,
+        res_data,
+        "priority",
+        routing_data,
+        globalState
+      );
+      if (should_continue)
+        should_continue = utils.should_continue_further(res_data);
+    });
+
+    it("retrieve-routing-call-test", () => {
+      let data = utils.getConnectorDetails("common")["priorityRouting"];
+      let req_data = data["Request"];
+      let res_data = data["Response"];
+      cy.retrieveRoutingConfig(req_data, res_data, globalState);
+      if (should_continue)
+        should_continue = utils.should_continue_further(res_data);
+    });
+
+    it("activate-routing-call-test", () => {
+      let data = utils.getConnectorDetails("common")["priorityRouting"];
+      let req_data = data["Request"];
+      let res_data = data["Response"];
+      cy.activateRoutingConfig(req_data, res_data, globalState);
+      if (should_continue)
+        should_continue = utils.should_continue_further(res_data);
+    });
+
+    it("payment-routing-test", () => {
+      let data =
+        utils.getConnectorDetails("stripe")["card_pm"]["No3DSAutoCapture"];
+      let req_data = data["Request"];
+      let res_data = data["Response"];
+      cy.createConfirmPaymentTest(
+        fixtures.createConfirmPaymentBody,
+        req_data,
+        res_data,
+        "no_three_ds",
+        "automatic",
+        globalState
+      );
+      if (should_continue)
+        should_continue = utils.should_continue_further(res_data);
+    });
+
+    it("retrieve-payment-call-test", () => {
+      cy.retrievePaymentCallTest(globalState);
+    });
+  });
+
+  context("Routing with adyen as top priority", () => {
+    before("seed global state", () => {
+      cy.task("getGlobalState").then((state) => {
+        globalState = new State(state);
+      });
+    });
+
+    after("flush global state", () => {
+      cy.task("setGlobalState", globalState.data);
+    });
+    it("retrieve-mca", () => {
+      cy.ListMCAbyMID(globalState);
```

Comment:
Even here and everywhere we used

Reply:
Also, rename that in commands to follow PascalCase convention.